config/jenkins-jobs_infra.ci.jenkins.io.yaml

```
@@ -509,11 +509,24 @@ jobsDefinition:
             description: "docs.jenkins.io File Share Service Principal Writer"
             subscriptionId: "${JENKINSINFRA_AZURE_PRIMARY_SUBSCRIPTION_ID}"
             tenant: "${JENKINSINFRA_AZURE_TENANT_ID}"
+      stats.jenkins.io:
+        name: stats.jenkins.io PR previews
+        allowUntrustedChanges: true
+        jenkinsfilePath: Jenkinsfile_previews
+        enableGitHubChecks: true
+        disableTagDiscovery: true
+  website-production-jobs:
+    name: Website Production Jobs
+    description: "Folder hosting all the Website jobs dedicated to deployment in production"
+    kind: folder
```

```suggestion
    kind: folder
    credentials:
      netlify-auth-token:
        description: "Auth token to communicate with netlify"
secret: "${NETLIFY_AUTH_TOKEN}"
```

=> Sorry, I missed it on the first review. But you need this credential to allow preview

---

This job doesn't contain any Netflify preview stage,it doesn't need this credentials.

---

🤦 sorry for the noise, my suggestion above is nonsense